dts/bindings/gpio/ite,it8xxx2-gpio.yaml

```diff
@@ -18,6 +18,11 @@ properties:
     label:
       required: true
 
+    index:
+        type: int
+        required: true
+        description: GPIO group index
```

Comment:
Please expand this, it needs a lot more detail. Index into what? Please be specific so people know what to set here

Reply:
Done